### PR TITLE
Fix typos in column names

### PR DESCRIPTION
**Pull request by c-w**
It'd be a bit confusing to have both `eventime` and `eventtime` as column names so better to normalize on one of them.

### CODE REVIEW

**Review by erikschlegel — APPROVED** [review]
LGTM

**Comment by c-w** [comment]
Will merge once this one is approved too https://github.com/CatalystCode/project-fortis-spark/pull/86

**Pull request merged**
